src/pages/index.tsx

```diff
@@ -320,9 +247,8 @@ const IndexPage: React.SFC<IndexPageProps> = props => {
       </IndexLayout>
     );
   }
-  const mostRecentPost = data.posts.edges[0].node;
-  const posts = data.posts.edges.filter((_, idx) => idx > 0 && idx < 10);
-  const isNextPageExist = data.posts.edges.length > 10;
+  const posts = data.posts.edges.filter((_, idx) => idx < 9);
```

Comment:
이 부분 이렇게 줄여도 괜찮지 않을까요?
```suggestion
  const posts = data.posts.edges.slice(0, 9)
```